drake/CMakeLists.txt

```diff
@@ -143,6 +143,11 @@ endif()
 
 enable_testing()
 include(CTest)
+find_package(GTest REQUIRED)
+include_directories(${GTEST_INCLUDE_DIRS})
+if (GTEST_FOUND)
+  set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -pthread")
```

Comment:
you are adding -pthread to every link line with this.  is there a more selective way?  some other libs include it via their pkgconfig